src/test/java/com/microsoft/sqlserver/jdbc/SQLServerConnectionTest.java

```diff
@@ -569,7 +572,7 @@ public void testClientConnectionId() throws Exception {
             } catch (SQLException e) {
                 assertEquals(e.getMessage(), TestResource.getResource("R_connectionIsClosed"),

```

Comment:
expected is 1st param

Reply:
Not a part of this PR.

Reply:
yeah, but it's in the same place, why not fix?